app/src/main/java/fr/neamar/kiss/dataprovider/AppProvider.java

```diff
@@ -142,17 +140,13 @@ public void requestResults(String query, Searcher searcher) {
                 continue;
             }
 
-            matchInfo = fuzzyScore.match(pojo.normalizedName.codePoints);
-            match = matchInfo.match;
-            pojo.relevance = matchInfo.score;
+            FuzzyScore.MatchInfo matchInfo = fuzzyScore.match(pojo.normalizedName.codePoints);
```

Comment:
I have some memories of long days spent looking at flamegraphs back in 2018 and noticing declaring the variable outside of the loop actually had a performance impact. Might have been fixed with the new generations of JDK

Reply:
I just compared generated bytecode of these versions: there is no difference when applying this change.
I think it's already optimized because nothing was happening with these fields outside the loop.